react/components/LoginAsCustomer.tsx

```diff
@@ -32,16 +43,18 @@ const LoginAsCustomer = ({
 }: Props) => {
   const handleHeaderRendering = useCallback(
     () => (
-      <div className="flex items-center c-on-base--inverted">
+      <div className={`${handles.loginButton} flex items-center c-on-base--inverted`}>
```

Comment:
Being nitpicky: `handles` is defined after this method. Defining it before makes it clearer where it comes from.